src/ble/BleLayer.cpp

```diff
@@ -67,8 +67,8 @@
 #include <lib/support/logging/CHIPLogging.h>
 
 // Magic values expected in first 2 bytes of valid BLE transport capabilities request or response:
-#define CAPABILITIES_MSG_CHECK_BYTE_1 'n'
-#define CAPABILITIES_MSG_CHECK_BYTE_2 'l'
+#define CAPABILITIES_MSG_CHECK_BYTE_1 0x65
```

Comment:
I think in the spec it's defined per-bit so you can use c++14 binary literal `0b1100101`.